api.js

```diff
@@ -56,7 +61,17 @@ Api.prototype._runFile = function (file) {
 		args.push('--sorted');
```

Comment:
Sidenote: I think it's time we simplify argument passing to the forked process. Too much code now just proxying CLI args.

@jamestalmage @vdemedes I propose we just JSON.stringify() the data we want to pass instead of using CLI args. Thoughts?